src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER_EXCLUSIVE = 101;
+
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(),
+                new Random().nextInt(MAX_BALL_NUMBER_EXCLUSIVE));
```

Comment:
You are creating a new instance of Random every time the getRandomBall() method is called, which is unnecessary. You should create a single instance of Random and reuse it.